tests/test_queries.py

```diff
@@ -290,13 +293,13 @@ def test_LimitAtBatchThresholdEdge(self):
         res = yield self.coll.find(limit=100)
         self.assertEqual(len(res), 100)
 
-        yield self.coll.drop(safe=True)
+        yield self.coll.drop()
```

Comment:
In case anyone is wondering, pymongo never supported safe in drop and drop_collection... neither did we, it was just sucked up into **kwargs and ignored later on.
http://api.mongodb.org/python/current/api/pymongo/collection.html#pymongo.collection.Collection.drop